docs/user-guides/operations.md

```diff
@@ -0,0 +1,148 @@
+# Operations
+
+This page introduces best practices for operating a Vald cluster.
+
+
+
+## Deployment
+
+### Kubernetes cluster
+
+Since Vald agents hold vector data on their memory, unexpected disruption or eviction of agents may cause loss of indices.
+Also, disruption or deletion of worker nodes that have Vald agents may cause loss of indices.
+To minimize the loss of indices, it is better to increase number of nodes and pods.
```

Comment:
```suggestion
Since Vald agents stores vector data on their memory space, unexpected disruption or eviction of agents may cause loss of indices.
Also, disruption or deletion of worker nodes that have Vald agents may cause loss of indices.
If you need to prevent low accuracy effects caused by indices loss, it is better to increase the number of nodes and pods.
```